lib/vmdb/global_methods.rb

```diff
@@ -17,6 +17,15 @@ def is_integer?(n)
       true
     end
 
+    # Check to see if a field contains a valid integer
+    def is_array?(n)
```

Comment:
@gmcculloug - In line with my first comment: If we keep this method - I think we should rename it to `array?` and possibly do a follow up PR to clean up the names of the other `is_` methods in this file.